test-it/karma/http/HttpGet.Test.js

```diff
@@ -396,8 +393,13 @@ describe('Performing a GET request', function () {
             expect(responseHeader.headers).toBeNonEmptyObject();
 
             // body
-            var responseBody = textFormat.normalizeLineEndings(viPathParser('body'));
-            expect(responseBody).toBe(bodyReadAsString);
+            // Body bytes: '\xCB\xD2\x7C\x42\x00\xA9\x78\xC2';
+            // var bodyReadAsString = '\uFFFD\uFFFD\x7C\x42\x00\uFFFD\x78\uFFFD';
+            // var responseBody = textFormat.normalizeLineEndings(viPathParser('body'));
+
+            // Httpbin seed not stable between Python versions. See https://github.com/postmanlabs/httpbin/issues/598
+            // Need an alternate endpoint to test null values in responses. See https://github.com/postmanlabs/httpbin/issues/599
+            // expect(responseBody).toBe(bodyReadAsString);
```

Comment:
Disabled assertion because result is dependent on python version. Filed bug with httpbin related to this behavior.

Reply:
Interesting. Glad you filed an issue with httpbin. Should we also create an issue in Vireo and reference the httpbin issue in that? Or, leave a TODO in the comments referencing the httpbin issue?

Reply:
Created https://github.com/ni/VireoSDK/issues/666 to track the disabled test